tests/data-system-tests.ipynb

```diff
@@ -159,8 +151,8 @@
    "outputs": [],
    "source": [
     "# Test endpoints\n",
-    "assert maap._SEARCH_GRANULE_URL == f'https://api.{env_prefix}maap-project.org/api/cmr/granules'\n",
-    "assert maap._SEARCH_COLLECTION_URL == f'https://api.{env_prefix}maap-project.org/api/cmr/collections'\n",
+    "assert maap._SEARCH_GRANULE_URL == f'https://{maap_host}/api/cmr/granules'\n",
```

Comment:
Minor but maybe rename this notebook to "test-cmr.ipynb" or "test-search.ipynb" to match the other notebook and make it more clear what is being tested in this file